[2020-05-14 17:51:04] console.ERROR: Error thrown while running command "'cahe:clear'". Message: "An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'db_user'@'localhost' (using password: YES)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'db_user'@'localhost' (using password: YES) at /var/www/html/Zartisan_prod/zartisanV2/Symfo/zartisan/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'db_user'@'localhost' (using password: YES) at /var/www/html/Zartisan_prod/zartisanV2/Symfo/zartisan/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'db_user'@'localhost' (using password: YES) at /var/www/html/Zartisan_prod/zartisanV2/Symfo/zartisan/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'cahe:clear'","message":"An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'db_user'@'localhost' (using password: YES)"} []
